### PR TITLE
[TypeDeclaration] Add nullable param from null compare on StrictStringParamConcatRector

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
Fixed 🎉 /cc @staabm

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba let's merge to have faster feedback to test ;)

File: rules-tests/TypeDeclaration/Rector/ClassMethod/StrictStringParamConcatRector/Fixture/nullable_param_from_null_compare.php.inc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictStringParamConcatRector\Fixture;
+
+class NullableParamFromNullCompare
+{
+    public static function getOrderBy($orderby, $direction = 'ASC', $default = '')
+    {
+        if (null !== $orderby && ! empty($orderby)) {
```
Comment (staabm) [inline rules-tests/TypeDeclaration/Rector/ClassMethod/StrictStringParamConcatRector/Fixture/nullable_param_from_null_compare.php.inc:9]: will `is_null()` calls also be covered?

Comment (samsonasik) [reply rules-tests/TypeDeclaration/Rector/ClassMethod/StrictStringParamConcatRector/Fixture/nullable_param_from_null_compare.php.inc:9]: Yes, I tried:

```php
if (! is_null($orderby) && ! empty($orderby)) {
     return $orderby.' '.$direction;
}
```

and it also changed to nullable string `?string` 👍

**Pull request merged**